docs/guide/README.md

```diff
@@ -11,6 +11,7 @@ All data is sent and received as JSON.
 
 | API Version | Release Date | Software Version |
 |---|---|---|
+|3.7.0|2023-03-29|[3.51.0](https://api.arable.cloud/api/v2/doc#section/Changelog/3.51.0)|
```

Comment:
@vasavi-iot We don't need to change the tag version here(`3.51.0`), it refers to the [Changelog tag](https://github.com/Arable/api-user/blob/develop/CHANGELOG.md#3510) in api-user when all the major features and bug fixes in.

Reply:
ok